lameenc.c

```diff
@@ -48,6 +48,7 @@ static PyObject* Encoder_new(PyTypeObject *type, PyObject *args, PyObject *kwds)
         }
         lame_set_num_channels(self->lame, 2);
         lame_set_in_samplerate(self->lame, 44100);
+        lame_set_out_samplerate(self->lame, 0);  // 0 is the default in lame
```

Comment:
If this is the default already, there's no need to set it.

Reply:
Ah nice, will remove!